.github/workflows/unit-test.yml

```diff
@@ -76,6 +76,8 @@ jobs:
       - name: Build 🔧
         run: |
           npm run compile
+          # run additional compilation variants
```

Comment:
This is going to duplicate compile many many times because each package you run compile in also compiles its dependencies. This is why the compile is done from `npm run compile` to ensure everything is only compiled a single time.

Reply:
The time spent is about 6minutes compared to about 2minutes before. And the full-compilation only happens in browser-tests -- Node.js tests only run the root tsconfig.json build.

Since these products are going to be released, we need to make sure the releasing variant is working so I think this is acceptable, WDYT?